src/main/ts/rules/PreferFun.ts

```diff
@@ -0,0 +1,75 @@
+import { Rule } from 'eslint';
+import { ArrowFunctionExpression, Expression, FunctionExpression, Literal, Statement } from 'estree';
+
+export const preferFun: Rule.RuleModule = {
+  meta: {
+    type: 'problem',
+    docs: {
+      description: 'Prefer using the Katamari Fun helper functions, instead of inline functions.'
+    },
+    messages: {
```

Comment:
If someone can think of clearer error messages here, please let me know and I'll happily change them.